app/src/main/res/values-de/strings.xml

```diff
@@ -404,6 +404,18 @@ Bitte melden Sie auftretende Probleme via GitHub.</string>
   <string name="syncthing_disabled_change_settings">Einstellungen ändern</string>
   <!--Button text on the "syncthing disabled" dialog, used as menu item to stop syncthing service if "always_run_in_background" is true-->
   <string name="exit">Beenden</string>
+  <!-- Reasons for "syncthing disabled" dialog -->
```

Comment:
Yeah, I was afraid of something like that. I checked the history of the file and saw manual commits outside of transifex so I hoped it would be OK. I'll simply remove the file from the push and see that it'll be translated on transifex instead.